server/controllers/auth.rb

```diff
@@ -11,8 +11,12 @@ class AuthController < ApplicationController
     if params[:limitted]
       requires_login!
       params[:matching] ||= ""
-      User.where(Sequel.expr(active: true) & Sequel.like(:username, '%' + params[:matching] + '%'))
-        .to_json(root: true, :only => [:username, :id])
+      User.where(Sequel.expr(active: true) & (
+          Sequel.ilike(:username, '%' + params[:matching] + '%') |
+          Sequel.ilike(:name, '%' + params[:matching] + '%') |
+          Sequel.ilike(:email, params[:matching])
+        ))
+        .to_json(root: true, :only => [:id, :username, :name, :avatar_url])
```

Comment:
Can you switch the `:only => [...]` part to `:only => User.public_attrs` and add `:name` and `:avatar_url` to the `User.public_attrs` array.